lib/boombox/pipeline.ex

```diff
@@ -117,6 +117,31 @@ defmodule Boombox.Pipeline do
     proceed(ctx, state)
   end
 
+  @impl true
+  def handle_child_playing(:udp_source, ctx, state) do
```

Comment:
I think it should be fine to use `handle_child_setup` here, it shouldn't matter much but `handle_child_setup` is called slightly earlier that `handle_child_playing`